tests/adapters_t/sites_t/test_kubernetes.py

```diff
@@ -0,0 +1,232 @@
+from tardis.adapters.sites.kubernetes import KubernetesAdapter
+from tardis.exceptions.tardisexceptions import TardisError
+from tardis.utilities.attributedict import AttributeDict
+from tardis.interfaces.siteadapter import ResourceStatus
+from tests.utilities.utilities import async_return
+from tests.utilities.utilities import run_async
+
+from unittest import TestCase
+from unittest.mock import patch
+
+import logging
+from kubernetes_asyncio import client
+
+
+class TestKubernetesStackAdapter(TestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls.mock_config_patcher = patch(
+            "tardis.adapters.sites.kubernetes.Configuration"
+        )
+        cls.mock_config = cls.mock_config_patcher.start()
+        cls.mock_kubernetes_api_patcher = patch(
+            "tardis.adapters.sites.kubernetes.client.AppsV1Api"
+        )
+        cls.mock_kubernetes_api = cls.mock_kubernetes_api_patcher.start()
+
+    @classmethod
+    def tearDownClass(cls):
+        cls.mock_config_patcher.stop()
+        cls.mock_kubernetes_api_patcher.stop()
+
+    def setUp(self):
+        config = self.mock_config.return_value
+        test_site_config = config.TestSite
+        test_site_config.host = "https://172.16.254.1:6443"
```

Comment:
I assume this is a "fake" address, and not actually used. Can you please add an in-code comment to briefly explain the address?

Reply:
Was a fake adress changed it to 127.0.0.1 and added comment